packages/generate/src/utils/get-nonnullable-columns.ts

```diff
@@ -195,6 +199,10 @@ function getNodeName(node: LibPgQueryAST.Node | undefined): string {
     return concatStringNodes(node.ColumnRef.fields);
   }
 
+  if (node?.NullTest) {
+    return getNodeName(node.NullTest.arg);
+  }
+
```

Comment:
not 100% sure if these lines are needed

Reply:
I think it's okay for now to keep it as it is